server/models/pattern/core/pattern.go

```diff
@@ -149,6 +152,14 @@ func (p *Pattern) GenerateApplicationConfiguration() (v1alpha1.Configuration, er
 func (p *Pattern) GetServiceType(name string) string {
 	return p.Services[name].Type
 }
+func (p *Pattern) GenerateDynamicSchema() {
+	for _, svc := range p.Services {
+		if strings.HasPrefix(svc.Type, "$(#use ") { //Currently pattern import is supported only via URL
```

Comment:
The prefix string of `strings.HasPrefix(s, prefix string) bool` has a blank at the last. The next line of code does not have. So, here I suggest that we define a new variable as the parameters. It can avoid this situation.